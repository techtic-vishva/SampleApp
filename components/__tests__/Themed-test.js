import * as React from 'react'
import renderer from 'react-test-renderer'

import { useThemeColor, Text, View, KeyboardAvoidingView } from '../Themed'

it(`useThemeColor renders correctly`, () => {
	const tree = renderer.create(<useThemeColor />).toJSON()

	expect(tree).toMatchSnapshot()
})

it(`Text renders correctly`, () => {
	const tree = renderer.create(<Text>Snapshot test!</Text>).toJSON()

	expect(tree).toMatchSnapshot()
})

it(`View renders correctly`, () => {
	const tree = renderer.create(<View />).toJSON()

	expect(tree).toMatchSnapshot()
})

it(`KeyboardAvoidingView renders correctly`, () => {
	const tree = renderer.create(<KeyboardAvoidingView />).toJSON()

	expect(tree).toMatchSnapshot()
})
