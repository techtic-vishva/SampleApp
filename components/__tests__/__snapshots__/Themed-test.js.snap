// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KeyboardAvoidingView renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "#0b0805",
      },
      undefined,
    ]
  }
/>
`;

exports[`Text renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#fff",
        "fontFamily": "objektiv",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`View renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#0b0805",
      },
      undefined,
    ]
  }
/>
`;

exports[`useThemeColor renders correctly 1`] = `<useThemeColor />`;
